src/promptflow/promptflow/_sdk/entities/_trace.py

```diff
@@ -127,3 +136,19 @@ def _from_protobuf_object(obj: PBSpan, resource: typing.Dict) -> "Span":
             session_id=attributes[SpanAttributeFieldName.SESSION_ID],
             parent_span_id=parent_span_id,
         )
+
+
+class Trace:
```

Comment:
Looks like we are going to name this line run

Reply:
renamed